#!/bin/bash 

# patch-it - Companion mini-script to pull-original, this basically does the auto-patch step of patch-update, only manually. 
Intentionally dumb.
# $1 should be the full atom plus the ebuild without extension. e.g. www-client/mozilla-firefox/mozilla-firefox-3.6.7
SOURCE_DIR="/usr/portage"
PATCH_DIR="doc/patches"

echo Updating $1 ...
diff -U 1 $SOURCE_DIR/$1.ebuild ./$1.ebuild > $PATCH_DIR/$1.ebuild.patch

